Mango API/src/com/infiniteautomation/mango/rest/latest/UserCommentRestController.java

```diff
@@ -113,8 +114,10 @@ public ResponseEntity<UserCommentModel> createNewUserComment(
 
         //Assign a userId if there isn't one
         if(model.getUserId() <= 0){
```

Comment:
This seems odd to me, wouldn't this allow anyone to post a comment as any other user?

Reply:
There is also delete/update/query methods which is not used by UI.
I think initial requirement of this feature and current usage is completely different.